### PR TITLE
Updated test 0035

### PR DESCRIPTION
**Pull request by dbooth-boston**

### CODE REVIEW

**Comment by dbooth-boston** [comment]
Somehow i didn't manage to delete that .swp file.  Silly git.

**Comment by glenna-m** [comment]
I think we should probably squash this into 1 or 2 commits, not 7 before merging.  The history is getting really noisy and hard to follow with this development pattern.   The way to do that is documented here: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by dbooth-boston** [comment]
ok, i'll try to break it into two.

File: RDF-Pipeline/t/run-test.perl [hunk 1]
```diff
@@ -94,6 +96,29 @@
...
+	last;
+	}
+      warn "WARNING: $tDir \nhas a pipeline.ttl with the same modification time as the one \nin $otherTestDir\nThis may cause the framework to not notice a pipeline.ttl change\nwhen going from one test to another.  Attempting a fix by touching \nthe one in $tDir ...\n";
+      my $qf = quotemeta($pipelineFile);
+      !system("touch $qf") || die;
+      sleep 1; 	# In case we need to do it on another file later
+      warn " ... done touching.\n";
+      }
+    }
+
```
Comment (glenna-m) [inline RDF-Pipeline/t/run-test.perl:121]: Would it be possible to ensure different modified times stamps in the copy of the setup files process?  Does this need to be a completely separate step? 

If it does need to be a separate step, should this be made into a function for more clear encapsulation of this feature?

Comment (dbooth-boston) [inline RDF-Pipeline/t/run-test.perl:121]: I don't know if it needs to be a separate step.  I was concerned that some tests might run faster than the modification time is detectably updated, but I don't actually know if that is possible or not.
